README.md

```diff
@@ -1,2 +1,218 @@
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [HelloMicroverse] <a name="about-project"></a>
+
+> This project is about a excercise to understand Microverse tools
+
+**[Hello Microverse]** is a project to practice whit the Microverse tools
+
+## 🛠 Built With <a name="built-with">Visual Code Editor</a>
+
+### Tech Stack <a name="tech-stack">HTML/CSS</a>
+
+> HTML & CSS is markup metodology to undertand how is the web structured.
+
+<details>
+  <summary>Tool</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">Visual Code Editor</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Tool</summary>
+  <ul>
+    <li><a href="https://www.w3.org/standards/">Client PC - Web Browser</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.w3.org/standards/">No Database requiered</a></li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Tool used on Microverse
+
+- **[Visual Studio Code]**
+- **[GitBash]**
+- **[HTML]**
+- **[CSS]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://larryivc.github.io/HelloMicroverse/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+This repo is a example of the use git, gitHub and Linterns.
+
+  - You need to make your own copy on your pc 💻
+  - Apply the HTML and CSS.
+  - Install Linters
+  - Sincronize with a <a href="https://github.com/">HitHub</a> 
+
+### Prerequisites
+
+In order to run this project you need:
+
+  - Web Browser
+  - Internet connection
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git git@github.com:LarryIVC/HelloMicroverse.git
+```
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+  cd my-project
+  start ./index.html
+```
+### Usage
+
+To run the project, execute the following command:
+
+
+Example command:
+
+```sh
+  start ./index.html
+```
+### Run tests
+
+To run tests, run the following command:
+
+
+Example command:
+
+```sh
+  start ./index.html
+```
+### Deployment
+
+You can deploy this project using a browser:
+
+
+Example:
+
+```Chrome index.html
+```
```

Comment:
[Optional] You can delete this section as it is irrelevant to this project👍. Please note that in future projects where you are required to test and deploy, you will need to have these sections with relevant instructions